Selenium.Axe/Selenium.Axe/Resources/scan.js

```diff
@@ -0,0 +1,23 @@
+﻿var callback = arguments[arguments.length - 1];
+
+// arguments[0] will have this script content
```

Comment:
This comment is confusing; it makes me expect that `arguments[0]` would be the contents of this script (`scan.js`), but the usage suggests that this is not the case

Reply:
forgot to remove this.

---
In reply to: [311814840](https://github.com/devaradhanm/SeleniumAxeDotnet/pull/1#discussion_r311814840) [](ancestors = 311814840)